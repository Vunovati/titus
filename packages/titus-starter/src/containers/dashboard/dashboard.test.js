import React from 'react'
import { render } from 'react-testing-library'
import Dashboard from './dashboard'

describe('<Dashboard />', () => {
  it('displays a basic dashboard', () => {
    const { container } = render(<Dashboard />)

    expect(container).toMatchSnapshot()
  })
})
