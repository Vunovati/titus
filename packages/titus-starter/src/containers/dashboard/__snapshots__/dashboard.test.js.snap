// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> displays a basic dashboard 1`] = `
<div>
  <div>
    I'm a Dashboard
  </div>
</div>
`;
